src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/TypeScript/html-report/index.html

```diff
@@ -8,7 +8,7 @@
 
 <head>
   <meta charset="UTF-8" />
-  <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline'">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline' image-src * data:;">
```

Comment:
```suggestion
  <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline'; img-src * data:;">
```
A semicolon is required before the term and it should be `img-src`

Reply:
Thanks for catching! Should be fixed now.